geoalchemy2/comparator.py

```diff
@@ -92,15 +94,15 @@ def __getattr__(self, name):
         func_ = _FunctionGenerator(expr=self.expr)
         return getattr(func_, name)
 
-    def intersects(self, other):
+    def intersects(self, other: _COMPARATOR_INPUT_TYPE) -> ColumnElement:
         """The ``&&`` operator. A's BBOX intersects B's."""
         return self.operate(INTERSECTS, other, result_type=sqltypes.Boolean)
 
-    def overlaps_or_to_left(self, other):
+    def overlaps_or_to_left(self, other: _COMPARATOR_INPUT_TYPE):
```

Comment:
Can this return type be annotated as `ColumnElement`?

Reply:
Oh yes you're right, I forgot this one. Thanks